compiler/infra/OMRCfg.cpp

```diff
@@ -326,8 +326,7 @@ TR_Structure *
 OMR::CFG::invalidateStructure()
    {
    setStructure(NULL);
-   _structureRegion.~Region();
-   new (&_structureRegion) TR::Region(comp()->trMemory()->heapMemoryRegion());
+   TR::Region::reset(_nodeRegion, comp()->trMemory()->heapMemoryRegion());
```

Comment:
I think this is supposed to be

```suggestion
   TR::Region::reset(_structureRegion, comp()->trMemory()->heapMemoryRegion());
```